docs/terms

```diff
@@ -0,0 +1,17 @@
+Terms of Use
+=====================
+
+CERN Analysis Preservation facilitates the collaborative nature of the LHC collaborations. The access restrictions are those selected by the respective LHC collaboration. Do not share materials outside the foreseen access restrictions or otherwise circumvent CAP’s access restrictions. Many materials that can be found through the service might be work in progress, so they should be used with caution. According to collaboration practices, it is not allowed to publish materials openly that are not approved by the respective committees within the collaboration. 
```

Comment:
This should be terms.rst

Reply:
OK. 
Terms of Use is in the menu as `terms`.
The reason I didn't do a folder, is because I looked at the reana documentation and it looked much cleaner like that (https://github.com/reanahub/reana-client/tree/master/docs).

Reply:
Ok it's up to you

This is how my menu looks like 
<img width="221" alt="screen shot 2018-06-15 at 14 39 40" src="https://user-images.githubusercontent.com/13986265/41468405-2dcd2e22-70aa-11e8-8c8b-212f9d2f26f2.png">

Reply:
Yeah, it gets the titles from the pages themselves, I can change those as well. But I don't know why the terms are not showing..